spring-cloud-alibaba-examples/seata-example/storage-service-provider/src/main/java/com/alibaba/cloud/examples/api/impl/StorageDubboServiceImpl.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2013-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.cloud.examples.api.impl;
+
+import com.alibaba.cloud.examples.api.StorageDubboService;
+import io.seata.core.context.RootContext;
+import org.apache.dubbo.config.annotation.DubboService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * @author <a href="mailto:zhangqian9158@gmail.com">ZHANGQIAN</a>
+ */
+
+@DubboService
+public class StorageDubboServiceImpl implements StorageDubboService {
```

Comment:
Don't need to add dubbo contents in Spring Cloud Alibaba

Reply:
Dubbo content modified!